card/lib/generators/card/card_generator.rb

```diff
@@ -0,0 +1,33 @@
+# -*- encoding : utf-8 -*-
+
+class Card
+  module Generators
+    class CardGenerator < NamedBase
+      include Rails::Generators::Migration
+      extend ActiveRecord::Generators::Migration
+
+      source_root File.expand_path('../templates', __FILE__)
+
+      argument :mount_point, :required=>false
+      argument :prefix, :required=>false
```

Comment:
Support here for prefix on the table names, and a mount point for the engine, default to none at root.

The 'name' seems standard to all generator commands, so I use it to prefix the files you might merge with existing ones. (config/routes.rb is the one for now, but maybe Gemfile and others later.